Userland/Libraries/LibGfx/ImageFormats/JPEGXLLoader.cpp

```diff
@@ -1965,10 +1965,7 @@ static ErrorOr<void> read_lf_group(LittleEndianInputBitStream&,
         if (channel.hshift() < 3 || channel.vshift() < 3)
             continue;
 
-        // This code actually only detect that we need to read a null image
-        // so a no-op. It should be fully rewritten when we add proper support
-        // for LfGroup.
-        TODO();
+        dbgln("Fixme: Decode ModularLFGroup for channel: {}x{}(h:{}, v:{})", channel.width(), channel.height(), channel.hshift(), channel.vshift());
```

Comment:
Should we have a test that hits this? Else it's easy to forget about if it doesn't crash or fail but only prints a message.